types.go

```diff
@@ -230,6 +230,17 @@ func Availability(v AvailabilityValue) *AvailabilityValue {
 	return Ptr(v)
 }
 
+// BranchProtectionDefaults represents default Git protected branch permissions.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/groups.html#options-for-default_branch_protection_defaults
+type BranchProtectionDefaults struct {
+	AllowedToPush           []int `json:"allowed_to_push,omitempty"`
+	AllowForcePush          bool  `json:"allow_force_push,omitempty"`
+	AllowedToMerge          []int `json:"allowed_to_merge,omitempty"`
+	DeveloperCanInitialPush bool  `json:"developer_can_initial_push,omitempty"`
+}
```

Comment:
I prefer to have this struct defined below the `Settings` struct.

Reply:
This should be addressed now. Let me know if you would like it moved somewhere differently, I assumed you meant the bottom of the file, but I just recognized that you might have meant right below. I don't mind moving it to wherever you'd prefer.